types/webpack/webpack-tests.ts

```diff
@@ -895,3 +895,11 @@ profiling = new webpack.debug.ProfilingPlugin({ outputPath: './path.json' });
 configuration = {
     plugins: [profiling]
 };
+
+compiler.hooks.done.tap('foo', stats => {
+  if (stats.startTime === undefined || stats.endTime === undefined) {
+    throw new Error('Well, this is odd');
```

Comment:
It would be nice to add a more descriptive error message, no matter how uncommon running into it may be.

Reply:
It doesn't matter, this never gets executed, so nobody will have to try interpreting this error message outside of this context.